packages/babel-core/test/config-chain.js

```diff
@@ -6,6 +6,27 @@ import util from "util";
 import escapeRegExp from "lodash/escapeRegExp";
 import * as babel from "../lib";
 
+// "minNodeVersion": "10.0.0" <-- For Ctrl+F when dropping node 10
+const supportsESM = parseInt(process.versions.node) >= 12;
+
+const isMJS = file => file.slice(-4) === ".mjs";
```

Comment:
nit: `String#endsWith` is available since Node.js 4
```suggestion
const isMJS = file => file.endsWith(".mjs");
```

Reply:
can use https://nodejs.org/api/path.html#path_path_extname_path as well, dunno if it's any better, tho